challenge_9/python/sarcodian/src/challenge_9.py

```diff
@@ -0,0 +1,23 @@
+def square_n_sort(L):
+
+    L_square = []
+    L_sorted = []
+    count = len(L)
+
+    if L[0] >= 0:
+        for i in L:
+            L_square.append(i**2)
+        return L_square
+    
```

Comment:
is the first 'if' step necessary? it seems to me,, the later 'while' loop is enough to kill the problem.